templates/gallery.html

```diff
@@ -14,30 +14,38 @@ <h1 class="title text-center">&lt;{{ this.title }}/&gt;</h1>
+                                            <img src="https://live.staticflickr.com/65535/49714282878_8595887a5a_z.jpg" width="1801" height="0" alt="Sponsors - PyCon Colombia 2020">
+                                        </a>
                                     </div>
                                 </div>
                                 <div class="row">
                                     <div class="col-md-6 col-sm-12">
-                                        <a data-flickr-embed="true" data-header="true" data-footer="true"
-                                            href="https://www.flickr.com/photos/pyconcolombia/albums/72157708912672196" title="Talks"><img
-                                                src="https://live.staticflickr.com/65535/47996489897_bf5f43a9c1_o.jpg" width="1801" height="0"
-                                                alt="Talks"></a>
+                                        <a data-flickr-embed="true" data-header="true" data-footer="true" href="https://www.flickr.com/photos/pyconcolombia/albums/72157713680977338" title="Team - PyCon Colombia 2020">
+                                            <img src="https://live.staticflickr.com/65535/49714049348_8525499c4b_z.jpg" width="1801" height="0" alt="Team - PyCon Colombia 2020">
+                                        </a>
                                     </div>
                                     <div class="col-md-6 col-sm-12">
-                                        <a data-flickr-embed="true" data-header="true" data-footer="true"
-                                            href="https://www.flickr.com/photos/pyconcolombia/albums/72157708931689266" title="Sponsors"><img
-                                                src="https://live.staticflickr.com/65535/48004867262_aee6716765_o.jpg" width="1801" height="0"
-                                                alt="Sponsors">
+                                        <a data-flickr-embed="true" data-header="true" data-footer="true" href="https://www.flickr.com/photos/pyconcolombia/albums/72157713679835003" title="Keynotes - PyCon Colombia 2020">
+                                            <img src="https://live.staticflickr.com/65535/49714271156_c55ff57c93_z.jpg" width="1801" height="0" alt="Keynotes - PyCon Colombia 2020">
+                                        </a>
+                                    </div>
+                                </div>
+                                <div class="row">
+                                    <div class="col-md-6 col-sm-12">
+                                        <a data-flickr-embed="true" data-header="true" data-footer="true" href="https://www.flickr.com/photos/pyconcolombia/albums/72157713679835003" title="Keynotes - PyCon Colombia 2020">
+                                            <img src="https://live.staticflickr.com/65535/49714271156_c55ff57c93_z.jpg" width="1801" height="0" alt="Keynotes - PyCon Colombia 2020">
```

Comment:
@jonathanvm13 I think this album is duplicated

Reply:
done